src/tulip/ids/patch.ts

```diff
@@ -0,0 +1,16 @@
+export const patch = {
```

Comment:
Perhaps add some descriptive comments here to indicate that this is a work-around